src/test/java/junit/edu/umich/its/cpmtest/MigrationTaskServiceTest.java

```diff
@@ -128,4 +128,14 @@ public void testGoogleGroupsNamefromSiteIDWhenNoProperEmailIdIsInFeed(){
 		assertEquals("9bfee837-9b2c-4b6e-9e84-d77e9fbe10e9",googleGroupName);
 
 	}
+
+
+	@Test
+	public void testCheckForDotsInFilename(){
```

Comment:
Good Idea!, I have added `assertEquals("Happy",MigrationTaskService.replaceDotsInFileNameExceptFileExtention("Happy"));` and is working as expected:). I will update this once the review is complete